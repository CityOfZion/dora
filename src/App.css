#application-wrapper {
}

#router-container {
  display: flex;
  width: 100%;
  flex: 1;
}

#router-content-wrapper {
  display: flex;
  flex-direction: column;
  flex: 1;
  width: 100%;
}

.column-container {
  display: flex;
  flex-direction: column;
  flex: 1;
  width: 100%;
}

/* #inner-router-container {
  margin: 12px;
} */

.page-container {
  background-color: #1a2f34;
  display: flex;
  flex-direction: column;
  flex: 1;
  width: 100%;
  padding: 24px 0;
  /* padding: 12px; */
}
