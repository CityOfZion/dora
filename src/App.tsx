import React, { ReactElement } from 'react'

import './App.css'
import Home from './pages/home/Home'

function App(): ReactElement<void> {
  return (
    <div className="App">
      <Home />
    </div>
  )
}

export default App
