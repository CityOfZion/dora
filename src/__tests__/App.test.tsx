import React from 'react'
import configureStore from 'redux-mock-store'
import { Provider } from 'react-redux'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

import App from '../App'
import thunk from 'redux-thunk'

const middlewares = [thunk]
const mockStore = configureStore(middlewares)

test('renders without crashing', () => {
  const tree = shallow(
    <Provider
      store={mockStore({
        block: { isLoading: false, list: [] },
        transaction: { isLoading: false, list: [] },
      })}
    >
      <App />
    </Provider>,
  )

  expect(toJson(tree)).toMatchSnapshot()
})
