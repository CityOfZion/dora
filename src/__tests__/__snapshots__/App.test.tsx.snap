// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  id="application-wrapper"
>
  <div />
  <div
    id="router-content-wrapper"
  >
    <div
      id="Home"
    >
      <div
        id="neoscan-logo-container"
      >
        <img
          alt="neoscan-logo"
          id="neoscan-logo"
          src={Object {}}
        />
        <div
          id="welcome-text"
        >
          <h1>
             welcome to neoscan 
          </h1>
           
          <div
            id="welcome-text-underscore"
          >
            _
          </div>
        </div>
        <span>
          Your home for all NEO related blockchain information
        </span>
      </div>
      <div
        className="list-row-container"
      >
        <div
          className="list-wrapper"
        >
          <div
            className="label-wrapper"
          >
            <label>
              explore blocks
            </label>
          </div>
          <div
            className="data-list-container"
          >
            <div
              className="data-list"
              style={
                Object {
                  "gridTemplateColumns": "repeat(4, auto)",
                }
              }
            >
              <div
                className="data-list-column"
                style={
                  Object {
                    "borderRadius": "3px 0 0 3px",
                  }
                }
              >
                Index
              </div>
              <div
                className="data-list-column"
                style={Object {}}
              >
                Time
              </div>
              <div
                className="data-list-column"
                style={Object {}}
              >
                Transactions
              </div>
              <div
                className="data-list-column"
                style={Object {}}
              >
                Size
              </div>
            </div>
          </div>
        </div>
        <div
          className="list-wrapper"
        >
          <div
            className="label-wrapper"
          >
            <label>
              explore transactions
            </label>
          </div>
          <div
            className="data-list-container"
          >
            <div
              className="data-list"
              style={
                Object {
                  "gridTemplateColumns": "repeat(3, auto)",
                }
              }
            >
              <div
                className="data-list-column"
                style={
                  Object {
                    "borderRadius": "3px 0 0 3px",
                  }
                }
              >
                Transaction ID
              </div>
              <div
                className="data-list-column"
                style={Object {}}
              >
                Size
              </div>
              <div
                className="data-list-column"
                style={Object {}}
              >
                Time
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div />
</div>
`;
